### PR TITLE
[#178] Add hints about plug-in stanza placement to README.

### PR DESCRIPTION
**Pull request by cbcunc**
Added additional time-saving hints about the order of REI stanzas in `/etc/irods/server_config.json`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ To activate the plugin, add a new stanza to the "rule_engines" array within `ser
 }
 ```
 
-Adding the Python Rule Engine Plugin stanza above the default "irods_rule_engine_plugin-irods_rule_language" stanza will allow any defined Python rules to take precedence over any similarly named rules in the iRODS Rule Language.
+Adding the Python Rule Engine Plugin stanza above the default "irods_rule_engine_plugin-irods_rule_language" stanza will allow any defined Python rules to take precedence over any similarly named rules in the iRODS Rule Language. Also, placing the "irods_rule_engine_plugin-python-instance" stanza before the "irods_rule_engine_plugin-cpp_default_policy-instance" stanza will insure that any default C++ policies will not cancel out any similarly named Python rules.
```
Comment (alanking) [inline README.md:37]: Let's omit the word "Also" and change "insure" to "ensure"
```suggestion
Adding the Python Rule Engine Plugin stanza above the default "irods_rule_engine_plugin-irods_rule_language" stanza will allow any defined Python rules to take precedence over any similarly named rules in the iRODS Rule Language. Placing the "irods_rule_engine_plugin-python-instance" stanza before the "irods_rule_engine_plugin-cpp_default_policy-instance" stanza will ensure that any default C++ policies will not cancel out any similarly named Python rules.
```

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ When the plugin is freshly installed, a template file is located at `/etc/irods/
 
 # Default PEPs
 
-The example `core.py.template` file in this repository contains a Python implementation of all static policy enforcement points (PEPs), equivalent to the default `core.re` rulebase.
+The example `core.py.template` file in this repository contains a Python implementation of all static policy enforcement points (PEPs), equivalent to the default `core.re` rulebase. Again, placing the "irods_rule_engine_plugin-python-instance" stanza before the "irods_rule_engine_plugin-cpp_default_policy-instance" stanza in `/etc/irods/server_config.json` will insure that any default C++ policies will not cancel out any similarly named Python rules copied from the example `core.py.template` file.
```
Comment (alanking) [inline README.md:53]: Let's omit the word "Again" and change "insure" to "ensure":
```suggestion
The example `core.py.template` file in this repository contains a Python implementation of all static policy enforcement points (PEPs), equivalent to the default `core.re` rulebase. Placing the "irods_rule_engine_plugin-python-instance" stanza before the "irods_rule_engine_plugin-cpp_default_policy-instance" stanza in `/etc/irods/server_config.json` will ensure that any default C++ policies will not cancel out any similarly named Python rules copied from the example `core.py.template` file.
```

**Review by alanking — COMMENTED** [review]
Thanks for the PR. I had a couple of suggestions...

**Comment by cbcunc** [comment]
@alanking Thank you for the review and suggestions. Resolved by committing your suggestions.

**Review by alanking — COMMENTED** [review]
Very good, thanks!

Let's squash this down to one commit and we can get it in. Are you familiar with that process? Happy to do that for you or even walk through how that works. Please let me know your preference there if you can't do the squash yourself.

**Comment by cbcunc** [comment]
@alanking OK, all squooshed.

**Review by alanking — APPROVED** [review]
Thanks!

**Pull request merged**